client/post-editor/editor-diff-viewer/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getPostRevision,
+	normalizeForDisplay,
+} from 'state/posts/revisions/selectors';
+
+class EditorDiffViewer extends PureComponent {
+	render() {
+		return (
+			<div className="editor-diff-viewer">
+				<h3 className="editor-diff-viewer__title">
+					{ this.props.revision.title }
+				</h3>
+				{ this.props.revision.content }
+			</div>
+		);
+	}
+}
+
+EditorDiffViewer.propTypes = {
+	postId: PropTypes.number,
+	revision: PropTypes.object,
+	selectedRevisionId: PropTypes.number,
+	siteId: PropTypes.number,
+};
+
+export default connect(
+	( state, ownProps ) => ( {
+		revision: normalizeForDisplay( getPostRevision( state, ownProps.siteId, ownProps.postId, ownProps.selectedRevisionId ) ),
```

Comment:
Updated to:
```
revision: normalizeForDisplay(
    getPostRevision( state, ownProps.siteId, ownProps.postId, ownProps.selectedRevisionId )
),
```
which makes it consistent with the indentation used in `editor-revisions-list/index.jsx`.

Other solutions:
- `flow`-ing for 2 functions (and one with so many parameters) is probably overkill
- destructing AFAIK would mean updating `getPostRevision` to: `getPostRevision = ( state, { siteId, postId, revisionId} )` which means we can't use `createSelector` anymore because the second arg is effectively an object and we only accept `number`, `boolean` or `string` [0]

[0] https://github.com/Automattic/wp-calypso/blob/master/client/lib/create-selector/index.js#L17